### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [ ] New feature
 - [x] Bug fix
 - [ ] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the current behavior (link to any open issues here)?**

Typos https://github.com/jsoref/postgres-operator/actions/runs/7992649567/attempts/1#summary-21826635919

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

Fewer typos https://github.com/jsoref/postgres-operator/actions/runs/7992649696/attempts/1#summary-21826636457

**Other Information**:

### CODE REVIEW

**Review by tjmoore4 — CHANGES_REQUESTED** [review]
Overall this PR looks good, thank you for the submission. There is one change that we'll need due to a variable name update to allow our test suite to run.

**Review by tjmoore4 — APPROVED** [review]
Updates look good, thanks.

File: pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go [hunk 1]
```diff
@@ -414,7 +414,7 @@ type RepoStatus struct {
 	ReplicaCreateBackupComplete bool `json:"replicaCreateBackupComplete,omitempty"`
 
 	// A hash of the required fields in the spec for defining an Azure, GCS or S3 repository,
-	// Utilizd to detect changes to these fields and then execute pgBackRest stanza-create
+	// Utilized to detect changes to these fields and then execute pgBackRest stanza-create
```
Comment (benjaminjb) [inline pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go:417]: why not "utilized"? Or a "." at the end of line 416?

File: installers/olm/Makefile [hunk 2]
```diff
@@ -66,7 +66,7 @@ tools: ## Download tools needed to build bundles
 tools: tools/$(SYSTEM)/jq
 tools/$(SYSTEM)/jq:
 	install -d '$(dir $@)'
-	curl -fSL -o '$@' "https://github.com/stedolan/jq/releases/download/jq-1.6/jq-$$(SYSTEM='$(SYSTEM)'; \
+	curl -fSL -o '$@' "https://github.com/jqlang/jq/releases/download/jq-1.6/jq-$$(SYSTEM='$(SYSTEM)'; \
```
Comment (benjaminjb) [inline installers/olm/Makefile:69]: Where's this change come from?

Comment (jsoref) [reply installers/olm/Makefile:69]: https://github.com/stedolan/jq was moved to https://github.com/jqlang/jq -- see https://github.com/jqlang/jq/blob/master/NEWS.md#17

(I meant to flag that, sorry)

File: pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go (hunk 1 above)
Comment (jsoref) [reply pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go:417]: Yeah, this struck me as weird but I didn't have enough of an anything to be able to figure out what to do to it.

Between not being particularly familiar w/ the comment structure, or the code...

Comment (benjaminjb) [reply pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go:417]: Yeah, not your problem, I was just curious. Thx!

**Pull request merged**